common4j/src/main/com/microsoft/identity/common/java/providers/oauth2/OAuth2StrategyParameters.java

```diff
@@ -44,8 +46,16 @@ public class OAuth2StrategyParameters {
     @Nullable
     private final transient AbstractAuthenticationScheme mAuthenticationScheme;
 
+    // TODO preferably this would live in a dedicated NativeAuthOAuth2StrategyParameters class, but
+    // that would require adding generics to Authority.java
+    @Nullable
```

Comment:
Left a comment about Nullable elsewhere

Reply:
I think `Collections.emptyList` is made for this

Reply:
Using an empty list would be incorrect semantics. An empty list would mean that the challenge types should be sent as an empty string. A null field would mean that it should not be sent.